File: src/main/java/com/cocktailsguru/app/cocktail/domain/CocktailGlass.java
package com.cocktailsguru.app.cocktail.domain;

import lombok.Data;

import javax.persistence.*;

@Entity(name = "coctail_glass")
@Data
public class CocktailGlass {
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private Long id;

    @Column(nullable = false)
    private String name;
}

File: src/main/java/com/cocktailsguru/app/cocktail/domain/CocktailGlass.kt
package com.cocktailsguru.app.cocktail.domain

import javax.persistence.Entity
import javax.persistence.GeneratedValue
import javax.persistence.GenerationType
import javax.persistence.Id

@Entity(name = "coctail_glass")
data class CocktailGlass(
        @Id
        @GeneratedValue(strategy = GenerationType.IDENTITY)
        val id: Long,
        val name: String
)